ownCloud/Server List/ServerListTableHeaderView.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  ServerListTableHeaderView.swift
+//  ownCloud
+//
+//  Created by Matthias Hühne on 27.01.19.
+//  Copyright © 2019 ownCloud GmbH. All rights reserved.
+//
+
+import UIKit
+
+class ServerListTableHeaderView: UIView, Themeable {
+
+    var messageThemeApplierToken : ThemeApplierToken?
+    var textLabel : UILabel = UILabel()
+    
+    
+    override init(frame: CGRect) {
+        
+        super.init(frame: frame)
+        
+        Theme.shared.register(client: self, applyImmediately: true)
+        
+        textLabel.translatesAutoresizingMaskIntoConstraints = false
+        textLabel.text = "Locations".localized
+        textLabel.font = UIFont.boldSystemFont(ofSize: 24.0)
```

Comment:
@hosy can we use dynamic-type here? Seen in another PR that we started using it.

Reply:
Will be handled in a separate issue